pkg/ui/webpack.app.js

```diff
@@ -36,111 +36,114 @@ function shouldProxy(reqPath) {
 }
 
 // tslint:disable:object-literal-sort-keys
-module.exports = (distDir, ...additionalRoots) => ({
-  entry: ["./src/index.tsx"],
-  output: {
-    filename: "bundle.js",
-    path: path.resolve(__dirname, distDir),
-  },
+module.exports = (env) => {
+  let localRoots = [path.resolve(__dirname)];
+  if (env.dist === "ccl") {
+    // CCL modules shadow OSS modules.
+    localRoots.unshift(path.resolve(__dirname, "ccl"));
+  }
 
-  resolve: {
-    // Add resolvable extensions.
-    extensions: [".ts", ".tsx", ".js", ".json", ".styl", ".css"],
-    // Resolve modules from src directory or node_modules.
-    // The "path.resolve" is used to make these into absolute paths, meaning
-    // that only the exact directory is checked. A relative path would follow
-    // the resolution behavior used by node.js for "node_modules", which checks
-    // for a "node_modules" child directory located in either the current
-    // directory *or in any parent directory*.
```

Comment:
I actually think this comment is quite helpful.

Reply:
Done.